app/views/download/help_improve.html.haml

```diff
@@ -85,22 +74,149 @@
             eventLabel: 'Help us improve',
             nonInteraction: true
           })
+        }
 
+        var getElementValue = function(element) {
```

Comment:
Can we use JQuery for this? Or are we removing that as a dependency?

Reply:
The Design System has just removed jQuery as a dependency and Rails 5.1 has removed it from the default stack (I think you can still choose for it to be included in Rails). Whilst we still use it on the current frontend, I've avoided it here to try and avoid creating some technical debt.